vendor-extra/ApiClientBundle/src/Services/Article.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace Libero\ApiClientBundle\Services;
+
+use Libero\ApiClientBundle\ApiClientInterface;
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Promise\PromiseInterface;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+final class Article implements ApiClientInterface
```

Comment:
We've switched from using the term `Article` to the more generic `Content`. This should be called `ContentStoreClient`, and provide ways to access each of the content store API endpoints and methods (being worked on in https://github.com/libero/schemas/pull/1/, described in RAML).

Reply:
This can be done when #13 is merged.